sal_interfaces/AdamSensors/AdamSensors_Telemetry.xml

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="http://lsst-sal.tuc.noao.edu/schema/SALTelemetrySet.xsl"?>
+<SALTelemetrySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALTelemetrySet.xsd">
+    <SALTelemetry>
+        <Subsystem>AdamSensors</Subsystem>
+        <Version>3.8.2</Version>
+        <Description>Temperature readings from sensors connected to an ADAM-6024</Description>
+        <EFDB_Topic>AdamSensors_temperature</EFDB_Topic>
+        <item>
+            <EFDB_Name>temp_ch0</EFDB_Name>
+            <Description>Temperature reading on ADAM Analog Input 0</Description>
+            <IDL_Type>float</IDL_Type>
+            <Units>degree</Units>
```

Comment:
please use deg_C for tempereature. See astropy units on https://docs.astropy.org/en/stable/units/. Thanks